php-scoper/twig-inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare( strict_types=1 );
```

Comment:
any reason to add this?
I mean... it's great, but nobody uses it in our codebase